src/features/dfdElements/outputPortBehaviorValidation.ts

```diff
@@ -20,14 +20,14 @@ interface PortBehaviorValidationError {
  */
 @injectable()
 export class PortBehaviorValidator {
+    private static readonly ASSIGNMENT_REGEX =
+        /^Assignment\(\{(([A-Za-z0-9_\|]+(,\s*[A-Za-z0-9_\|]+)*)?)\};(\s*|!|TRUE|FALSE|\|\||&&|\(|\)|([A-Za-z][A-Za-z0-9_]*\.[A-Za-z][A-Za-z0-9_]*))+;\{(((([A-Za-z][A-Za-z0-9_\|]*)\.([A-Za-z][A-Za-z0-9_\|]*))+(,\s*(([A-Za-z][A-Za-z0-9_\|]*)\.([A-Za-z][A-Za-z0-9_\|]*)))*)?)\}\)+$/;
```

Comment:
This should be easier to read and performs the same check (minus allowing whitespace):
`/^Assignment\(\{(([A-Za-z0-9_]+([,|][A-Za-z0-9_]+)*)?)\};(!|TRUE|FALSE|\|\||&&|\(|\)|([A-Za-z0-9_]*\.[A-Za-z0-9_]*))+;\{(((([A-Za-z0-9_]*)\.([A-Za-z0-9_]*))+(,(([A-Za-z0-9_]*)\.([A-Za-z0-9_]*)))*)?)\}\)+$/`

Reply:
Should labels/flows be allowed to start with an _ ?

I see no reason for removing the whitespaces. Without them the Regex might get more simple but when defining the Assignments being able to use whitespaces does improve readability in the WebEditor.

Assignment({request,data,items};Encryption.Encrypted&&TRUE;{Sensitivity.Public}) is harder to read then 
Assignment({request, data, items}; Encryption.Encrypted && TRUE; {Sensitivity.Public})

Reply:
if you want to forbid starting with _ then every [A-Za-z0-9_] block in all regexes needs to have a block [A-Za-z] before hand which would blow to regex substantially.

We can keep whitespace if you like

Reply:
> if you want to forbid starting with _ then every [A-Za-z0-9_] block in all regexes needs to have a block [A-Za-z] before hand which would blow to regex substantially.

Yeah, you're right. I reduced the complexity but kept the whitespaces.